### PR TITLE
Update references to new 2.6 UI branches

### PR DESCRIPTION
**Pull request by richard-cox**
## Problem
- `rancher/dashboard` and `rancher/ui` have both now branched for generic 2.6 work
 
## Solution
- This change ensures the `rancher/rancher` 2.6 branch points to the correct builds
- These builds are used (when settings are default and the version ends in `-head`) as an alternative to the build embedded in to Rancher)
 
### Manual Testing
- Updated the settings in an existing rancher instance with the new values and ensured both UIs load correctly

### CODE REVIEW

**Review by MbolotSuse — APPROVED** [review]
Appears to pull a valid dashboard version, so LGTM

**Comment by richard-cox** [comment]
@stormqueen1990 / @jakefhyde Noticed you're down as capt'n for Q1-v2.6x. Michael has approved but requested an additional approval from a release captain.

**Review by jakefhyde — APPROVED** [review]

**Pull request merged**
